### PR TITLE
Add split polygon to bbox script

### PR DESCRIPTION
**Pull request by meomancer**
This will fix https://github.com/kartoza/tomorrownow_gap/issues/65

### CODE REVIEW

File: django_project/gap/factories/preferences.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+class PreferencesFactory(
+    BaseFactory[Preferences], metaclass=BaseMetaFactory[Preferences]
+):
+    """Factory class for Provider model."""
+
+    class Meta:  # noqa
+        model = Preferences
+
+    area_of_interest = factory.LazyFunction(
+        lambda: Polygon([
```
Comment (danangmassandy) [inline django_project/gap/factories/preferences.py:23]: @meomancer can you please change the default polygon to below? this one is proposed by Phillip.
Proposed BB:
Lat: -27deg S to +16deg N
Long: 21.8 deg E to 52 deg E

**Review by danangmassandy — CHANGES_REQUESTED** [review]

File: django_project/gap/models/preferences.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+from django.contrib.gis.db import models
+
+from core.models.singleton import SingletonModel
+
+
+class Preferences(SingletonModel):
+    """Preference settings specifically for gap."""
+
+    area_of_interest = models.PolygonField(
```
Comment (danangmassandy) [inline django_project/gap/models/preferences.py:17]: can we set default of area of interest to below? This bbox is from Phillip.
Proposed BB:
Lat: -27deg S to +16deg N
Long: 21.8 deg E to 52 deg E

**Review by danangmassandy — CHANGES_REQUESTED** [review]

**Comment by meomancer** [comment]
Hi @danangmassandy default is added
Selection_140

**Review by danangmassandy — APPROVED** [review]
Thanks @meomancer

**Pull request merged**